appcues/src/main/java/com/appcues/debugger/screencapture/AndroidTargetingStrategy.kt

```diff
@@ -21,9 +21,18 @@ internal data class AndroidViewSelector(
     var tag: String? = null,
     var resourceName: String? = null
 ) : ElementSelector {
+
     val isValid: Boolean
         get() = contentDescription != null || tag != null || resourceName != null
 
+    val displayName: String?
+        get() = when {
+            resourceName != null -> "RES/$resourceName"
+            tag != null -> "TAG/$tag"
+            contentDescription != null -> contentDescription
+            else -> null
+        }
```

Comment:
seems reasonable - maybe next step is to generate a snapshot from the test app and run it by Sam? Just to see if he has any opinions on the user facing aspect of this. Does he want to include `RES/` prefix for instance? I'm not sure if that has meaning to the end user in the builder or not.

Beyond this scope - do we want to try to identify if the type has `Button` or  `Label` or `Image` etc in the name and attempt to be making any edits to the name like `Button: $resource_name` ? I kind of hope not as I'm not sure how maintainable that is, but interested in his expectations.

Reply:
so as a first take I thought how we could differ the display name, my thought process here was to follow the same priority order we do for weight inside evaluateMatch. but the first problem is

1. we are basically in need of similar logic for both places to keep in sync.
2. if thats the case, changing weight or adding new selector types could change how the view is displayed.

things to ponder. I could not solve this one, I know its a good starting point but needs to evolve

Reply:
I noted a discussion topic for tomorrow just to get any input Sam has on desired UX in builder. Could you generate a snapshot with the examples as we have it here for him?

Reply:
as discussed with the team, looks like we can move on with this for now and revisit if necessary in the future